client/state/happychat/middleware.js

```diff
@@ -18,6 +25,16 @@ export const requestTranscript = ( connection, { getState, dispatch } ) => {
 	);
 };
 
+export const sendEventMessage = ( connection, { getState }, action ) =>{
```

Comment:
Maybe this should be more-specifically named something like `sendRouteEventMessage`?